params/params_joel.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 5
+    
+# e.g. "anas_krystallinus"    
+species.name <- "george "
+    
+# e.g. "red"    
+color <- "pink "
```

Comment:
Similar here. R won't recognise "pink " as a colour because of the trailing space.
```suggestion
color <- "pink"
```